packages/react/src/FormErrorList/useAddErrorCountToPageTitle.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react'
+
+export const useAddErrorCountToPageTitle = (
+  /** The list of error messages used to calculate the error count. */
+  errors: { id: string; label: string }[],
+  /** The text following the error count. */
+  suffix = {
+    plural: 'invoerfouten',
+    singular: 'invoerfout',
+  },
+) => {
+  const [documentTitle, setDocumentTitle] = useState<string>()
+
+  useEffect(() => {
+    setDocumentTitle(document.title)
+  }, [])
+
+  if (documentTitle) {
+    if (errors.length > 0) {
+      document.title = `(${errors.length} ${errors.length === 1 ? suffix.singular : suffix.plural}) ${documentTitle}`
```

Comment:
Now we have a ternary inside an if-statement nested inside an if-statement. Maybe add an `else` for the plural case to remove the ternary?

Reply:
Something like this you mean?

```
  if (errors.length === 1) {
    document.title = `(${errors.length} ${suffix.singular}) ${documentTitle}`
  } else if (errors.length > 1) {
    document.title = `(${errors.length} ${suffix.plural}) ${documentTitle}`
  } else {
    document.title = documentTitle
  }
```